release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -199,15 +199,15 @@ In the following tables, features are marked with the following statuses:
 --
 1. For {product-title} {product-version}, you must install the {product-title} cluster on a VMware vSphere version 7.0 Update 2 or later instance, including VMware vSphere version 8.0, that meets the requirements for the components that you use.
 --
-[discrete]
-=== Updating clusters deprecated and removed features
+//[discrete]
+//=== Updating clusters deprecated and removed features
```

Comment:
Not in the 4.13 notes, because no entries exist. The last feature mentioned was removed in 4.12?

![Screenshot from 2023-06-28 13-50-11](https://github.com/openshift/openshift-docs/assets/57954076/c5dd5aa1-ef1a-4be3-b23c-9e59aaf72e93)

Reply:
Good to note, I'm going to leave it just in case some things get added in this release. If we get closer to the release and there is nothing we can remove.